nyxtools/vector.py

```diff
@@ -256,9 +253,8 @@ def move(self):
             running = yield from bps.rd(self.running)
             elapsed = time.time() - t
 
-            if not running:
+            if not running or elapsed > self.timeout:
```

Comment:
It doesn't look like we can distinguish between a normal-looking completion and one that times out. Does the calling code need to check running time?

Reply:
I see that this was fixed later. Will resolve!